solr/contrib/jaegertracer-configurator/README.txt

```diff
@@ -0,0 +1,30 @@
+Welcome to Apache Solr Prometheus Exporter
+========
+
+Apache Solr Jaeger Tracer Configurator (solr-jaeger-tracer) provides a way for you to expose Solr's tracing to Jaeger.
+
+# Setup Jaeger Tracer Configurator
+
+Note that all library of solr-jaeger-tracer must be included in the classpath of all nodes then Jaeger tracer can be setup in `solr.xml` like this:
+
+```
+<tracerConfig name="tracerConfig" class="org.apache.solr.jaeger.JaegerTracerConfigurator">
```

Comment:
It'd be better if we drive samplePercentage dynamically through cluster properties. Imagine wanting to change the sample percentage and having to update solr.xml in zk and restarting all nodes. It would be quite cumbersome. Also, shouldn't this be collection specific? e.g. what if I want to sample a collection more aggressively than others?

Reply:
I'm not sure it's possible to have a `Tracer` per collection. If we want to sample a particular collection more aggressively than others, we should be able to perform our own *should-sample-check* and then manually start a trace, assuming OpenTracing supports this. This way we could differentiate the sample rate but still go through a single `Tracer`.

Reply:
I'm quite unsure about sample rate per collections. This kinda complex and requires more careful thoughts about it. Since there are APIs in Solr that does not tie specificly to any collection (Collections API, autoscaling, streaming expressions, etc).
It should be handled in another issue.